parcer-for-log2/Edy's_HW_3_Picnic_calc w_o debug mode.py

```diff
@@ -0,0 +1,69 @@
+import json
+import sys
+
+Data_file = "C:\\Users\\svetah\\Desktop\\py\\parcer-for-log2\\picnic_list.json"
+#meat_per_adult = float(0.5)   #kg # params for hardcoded instead of json
+#meat_per_child = float(0.2)  #kg # params for hardcoded instead of json
+#bear_per_adult = int(2) #bottle # params for hardcoded instead of json
+#bear_per_child = int(0) # params for hardcoded instead of json
+#soft_drink_per_adult = float(0.5) #litr # params for hardcoded instead of json
+#soft_drink_per_child = float(0.5) #litr # params for hardcoded instead of json
+#meat_price_per_kg # params for hardcoded instead of json
+#bear_price_per_botle # params for hardcoded instead of json
+#drink_price_per_botle # params for hardcoded instead of json
+
+
+
+elements_from_json = []  # empty dict for keep all relevant params from json file
+def read_from_file():
+    with open(Data_file, 'r') as basic_info:
+        read_data = json.load(basic_info)  #convert the json string to a dict
+        iner_dict = read_data['data_per_oreah'] #because of json is a list of lists , will use only params from data_per_oreh list
```

Comment:
You do not need here a temporary variable, just return it directly: return read_data['foo']
I would also rename "read_data" to something like: picnic_data
And note that you comment here is wron, there are no lists in your json.